Editor/Deployment/DeploymentSettings.cs

```diff
@@ -222,29 +230,16 @@ public void Restore()
             ScenarioIndex = 1; // Selects "Single-Region Fleet" Deployment Scenario by default
             BuildFilePath = null;
             BuildFolderPath = null;
-            GetSettingResponse response = _coreApi.GetSetting(SettingsKeys.DeploymentGameName);
-            GameName = response.Success ? response.Value : null;
-            GetSettingResponse scenarioIndexResponse = _coreApi.GetSetting(SettingsKeys.DeploymentScenarioIndex);
-
-            if (scenarioIndexResponse.Success)
-            {
-                int? index = SettingsFormatter.ParseInt(scenarioIndexResponse.Value);
-                ScenarioIndex = index ?? 0;
-            }
-
-            GetSettingResponse serverPathResponse = _coreApi.GetSetting(SettingsKeys.DeploymentBuildFolderPath);
-
-            if (serverPathResponse.Success)
-            {
-                BuildFolderPath = serverPathResponse.Value;
-            }
-
-            GetSettingResponse serverExePathResponse = _coreApi.GetSetting(SettingsKeys.DeploymentBuildFilePath);
-
-            if (serverExePathResponse.Success)
-            {
-                BuildFilePath = serverExePathResponse.Value;
-            }
+            
+            var index = SettingsFormatter.ParseInt(_coreApi.GetSetting(SettingsKeys.DeploymentScenarioIndex).Value);
+            ScenarioIndex = index ?? 1;
+            GameName = _coreApi.GetSetting(SettingsKeys.DeploymentGameName).Value;
+            BuildFolderPath = _coreApi.GetSetting(SettingsKeys.DeploymentBuildFolderPath).Value;
+            BuildFilePath = _coreApi.GetSetting(SettingsKeys.DeploymentBuildFilePath).Value;
+            LaunchParameters = _coreApi.GetSetting(SettingsKeys.LaunchParameters).Value;
+            BuildOperatingSystem = _coreApi.GetSetting(SettingsKeys.BuildOperatingSystem).Value;
+            FleetName = _coreApi.GetSetting(SettingsKeys.FleetName).Value;
+            BuildName = _coreApi.GetSetting(SettingsKeys.BuildName).Value;
```

Comment:
I know in #99 we introduced a `Settings` wrapper, which does the exact same functionality as this. Is there a reason we shouldn't be using that wrapper here too? This now creates two ways to do the same functionality, which often leads to inconsistencies and bugs

Reply:
I've made _coreApi use Settings wrapper, otherwise we would have to add new Settings objects everywhere

Reply:
Amazing, thank you!